test/functional/answers_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class AnswersControllerTest < ActionController::TestCase
```

Comment:
Ok I have added tests for answers and answers_like controller and they are working fine now.